packages/@okta/vuepress-theme-prose/const/navbar.const.js

```diff
@@ -570,6 +570,7 @@ export const reference = [
         ]
       },
       { title: "Error Codes", path: "/docs/reference/error-codes/" },
+      { title: "Hook best practices", path: "/docs/reference/hook-best-practices/" },
```

Comment:
Should this come after the hooks reference? 
Actually, why is Event Hook inside the Manage Okta Objects folder, but the Inline Hook and Hook best practices references not in there? Should all the hooks be combined (I don't know the answer to this unfortunately)

Reply:
All the individual Hook API references are outside the Manage Okta Objects folder (not sure why either). So, added best practices here as well (alphabetically). I believe this Reference navigation is being re-organized with a separate Hooks folder anyway.